packages/react/src/components/Header/HeaderAction/HeaderAction.jsx

```diff
@@ -94,21 +112,25 @@ const HeaderAction = ({ item, index, testID }) => {
             isExpanded={isExpanded}
             ref={menuButtonRef}
             index={index}
+            inOverflow={inOverflow}
           />
         ) : (
           // otherwise render a submenu type dropdown
           <HeaderActionMenu
             className={`${carbonPrefix}--header-action-btn`}
             key={`menu-item-${item.label}`}
             aria-label={item.label}
-            renderMenuContent={() => item.btnContent}
+            renderMenuContent={() =>
+              isExpanded ? <Close16 fill="white" description="close-icon" /> : item.btnContent
```

Comment:
Is there perhaps a carbon color that we should use here, instead of just white?
Also, the description should probably use i18n.

Reply:
Good call. Replaced with carbon white, and added description via i18n prop